c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -404,6 +404,7 @@ $(PACKAGE)/locale/%/LC_MESSAGES/%/$(PACKAGE)-client.po: $(PACKAGE)/locale/$(PACK
 $(OUTPUT_DIR)/build.js: .build/node_modules.timestamp .build/build.js
 	mkdir -p $(dir $@)
 	awk 'FNR==1{print ""}1' $(addprefix node_modules/, jquery/dist/jquery.min.js angular/angular.min.js angular-gettext/dist/angular-gettext.min.js angular-animate/angular-animate.min.js bootstrap/dist/js/bootstrap.min.js) .build/build.js > $@
+    sed -i '/^\/\/# sourceMappingURL=.*\.map$$/d' $@
```

Comment:
oops, I guess it's related to the file not being seen as a Makefile by my editor. I'll fix it.